lib/core/enhanceError.js

```diff
@@ -17,11 +17,11 @@ module.exports = function enhanceError(error, config, code, request, response) {
   }
 
   error.request = request;
-  error.response = response;
+  error.response=response;
   error.isAxiosError = true;
 
-  error.toJSON = function() {
-    return {
+  error.toJSON = function( ) {
+    return{
```

Comment:
format: style mismatch:
```suggestion
    return {
```

`3ea352dc` ␣ should be inserted at column 11.